gpkit/interactive/plot_sweep.py

```diff
@@ -0,0 +1,95 @@
+"Implements plot_sweep1d function"
+import matplotlib.pyplot as plt
+from ..exceptions import InvalidGPConstraint
+
+
+def assign_axes(var, posys, axes):
+    "Assigns axes to posys, creating and formatting if necessary"
+    if not hasattr(posys, "__iter__"):
+        posys = [posys]
+    if axes is None:
+        N = len(posys)
+        _, axes = plt.subplots(N, 1, sharex="col", figsize=(4.5, 3+1.5*N))
+        if N == 1:
+            axes = [axes]
+        format_and_label_axes(var, posys, axes)
```

Comment:
should the above line be out-dented by 4 spaces? It seems like this method does no assignment if `axes is not None`